consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/IbftBlockHeaderValidationRulesetFactoryTest.java

```diff
@@ -62,7 +62,7 @@ public void bftValidateHeaderPasses() {
         getPresetHeaderBuilder(2, proposerNodeKey, validators, parentHeader).buildHeader();
 
```

Comment:
Can you add a test that shows that the header validation passes when we have header that contains a base fee